scripts/data/syntec/create_data.py

```diff
@@ -0,0 +1,31 @@
+import json
+import pandas as pd
+from huggingface_hub import create_repo, upload_file
+from huggingface_hub.utils._errors import HfHubHTTPError
+
+### Process articles ###
+articles = pd.read_json("./syntec.json")
```

Comment:
I think we should put the files on Wikit github or just use the hugging face dataset without this script (or both) because we won't push the raw files so people won't be able to use this script

Reply:
This script was used to process/upload data the data to HuggingFace. It is not meant to be reused to reupload the data on huggingface. MTEB then uses the HF datasets. I just wanted to store the script somewhere but it is not really meant to be reused by anyone.